### ISSUE TITLE
Document file format is not selectable

### ISSUE THREAD

**Issue by frobel** [shopware/platform#7997]
### PHP Version

8.3

### Shopware Version

6.6.x

### Affected area / extension

Platform(Default)

### Actual behaviour

If the ACCESSABILITY_TWEAK is enabled, the feature file formats are available for document templates (invoices). Currently, the merchant cannot select the file format from the select list (PDF or HTML)

![Image](https://github.com/user-attachments/assets/3243ac8f-8cc0-4028-a46b-0772726b77b9)

### Expected behaviour

The file formats for invoices should be selectable and visible in the input box.

### How to reproduce

1. ACCESSABILITY_TWEAK feature flag is enabled
2. Create a new Document template via settings > documents 
3. Try to select file format PDF or HTML from the selection list
